### DISCUSSION TITLE
Bring back Windows 7/8.1's notification balloon tray for ExplorerPatcher-Windows 10 Taskbar

### DISCUSSION THREAD

**Discussion by TK50P** [valinet/ExplorerPatcher#3622] [Ideas]
On Windows 11 and using **Windows 10 (ExplorerPatcher)** taskbar, it doesn't show the balloon even I applied registry.
I'm looking for similar like this.
![image](https://github.com/user-attachments/assets/b9a834e4-b9ff-4b07-8b48-6fcae0190eaa)
Is there any workaround? or currently working?

**Comment by pyrates999** [comment]
what registry change did you make?

**Comment by TK50P** [comment]
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Software\Policies\Microsoft\Windows\Explorer]
"EnableLegacyBalloonNotifications"=dword:00000001

**Reply by pyrates999** [reply]
Try this:

1. open up task manager
2. kill explorer.exe
3. run explorer.exe

If it still doesn't work, try rebooting.  If that still doesn't allow it to work, then MS has disabled this with the current build you have installed of windows 11, sorry.

**Comment by TK50P** [comment]
However, When I used 23H2 and with Windows 10 (non-explorerpatcher) version, it worked.

**Reply by Anixx** [reply]
For me, does not.

**Comment by pyrates999** [comment]
Oh well in that case, please wait for a developer to respond.

**Comment by Amrsatrio** [comment]
The old balloon notifications are already implemented in the code, but I haven't tested the usual tricks to enable it.

**Comment by TK50P** [comment]
![image](https://github.com/user-attachments/assets/7d885ef1-39b6-4308-9ef7-2ceaff0bce5d)
I provided example of parsec. however, even with Windows 10 (ExplorerPatcher) taskbar, it doesn't show with balloon.
If you think the program doesn't support it, you are wrong.
![Screenshot 2024-09-03 204434](https://github.com/user-attachments/assets/c05f1a95-4791-47c6-902a-2bf07c575093)
It shows legacy balloon.

**Reply by Amrsatrio** [reply]
What's the Windows version of the above screenshot and the bottom screenshot?

**Comment by TK50P** [comment]
First Image: Windows 11 Version 24H2 (26100.1457) with Windows 10 Taskbar with ExplorerPatcher remastered
![image](https://github.com/user-attachments/assets/b4802201-8e18-4371-8c32-cdb265ca886f)

Second Image (Sorry for poor quality): Windows 11 Version 23H2 (22631.4037) with Windows 10 Taskbar without ExplorerPatcher remastered
![image](https://github.com/user-attachments/assets/322836e3-a68f-4f40-acbe-fb48da050d36)

**Reply by Anixx** [reply]
For me it does not work on Win11 23H2 with Win10 taskbar without EP.

**Comment by Anixx** [comment]
For me on Win11 23H2 with non-EP Win10 taskbar, this does not work with the registry trick.
